doc/en/weechat_plugin_api.en.adoc

```diff
@@ -15006,13 +15006,13 @@ see example below (supported only in WeeChat ≥ 0.4.2).
 def bar_item_new(name: str, build_callback: str, build_callback_data: str) -> str: ...
 
 # example (callback without "buffer" and "extra_info")
-def my_build_callback(data, item, window):
+def my_build_callback(data: str, item: str, window: str) -> str:
     return "my content"
 
 bar_item = weechat.bar_item_new("myitem", "my_build_callback", "")
 
 # example (callback with all arguments, for WeeChat ≥ 0.4.2)
-def my_build_callback2(data, item, window, buffer, extra_info):
+def my_build_callback2(data: str, item: str, window: str, buffer: str, extra_info: Dict[Any, Any]) -> str:
```

Comment:
Wasn't quite sure what to type `extra_info` to here, since it's always an empty dict, and I'm not allowed to use the `Dict` type without types for the key and value, so I went for `Any` for the key and value.

Reply:
This parameter `extra_info` is not used today (reserved for future use).

Anyway if one day it is used, as all hashtables in scripting API, both key and value will be type `str` so you can set this type.

Reply:
> This parameter `extra_info` is not used today (reserved for future use).

Yes, I saw, but I noticed that I did get a dict, so wanted to type it as such even though it's always empty.

> Anyway if one day it is used, as all hashtables in scripting API, both key and value will be type `str` so you can set this type.

Alright, changed.